### ISSUE TITLE
Displaynames longer than 16 characters throw an error

### ISSUE THREAD

**Issue by Nuubles** [2lstudios-mc/FlameCord#130]
Setting the name to be longer than 16 characters, for example with colors, will throw an error.

How to reproduce:
```
ProxiedPlayer player = ...;
player.setDisplayName("§2Some§aLong§eDisplay§5N")
```

**Comment by Luccboy** [comment]
The error is thrown because of this line:
https://github.com/2lstudios-mc/FlameCord/blob/d6bbda8e53f1bae006fd8ccfcdc30fa340423c76/Waterfall-Proxy-Patches/0004-1.7.x-support.patch#L1063
I don't know if it's safe to remove this. @linsaftw has to check this.

**Comment by linsaftw** [comment]
> The error is thrown because of this line:
> 
> https://github.com/2lstudios-mc/FlameCord/blob/d6bbda8e53f1bae006fd8ccfcdc30fa340423c76/Waterfall-Proxy-Patches/0004-1.7.x-support.patch#L1063
> 
> 
> I don't know if it's safe to remove this. @linsaftw has to check this.

Maybe making it an option?

**Comment by Luccboy** [comment]
> Maybe making it an option?
Yeah, this is probably the best solution.

**Comment by Nuubles** [comment]
> > > The error is thrown because of this line:
> > > https://github.com/2lstudios-mc/FlameCord/blob/d6bbda8e53f1bae006fd8ccfcdc30fa340423c76/Waterfall-Proxy-Patches/0004-1.7.x-support.patch#L1063
> > > 
> > > I don't know if it's safe to remove this. @linsaftw has to check this.
> > 
> > 
> > Maybe making it an option?
> 
> Well, if you remove that you can unsecure lot of things, and even brake client i thing. Also, allow +16 args things xD, making it toggleable is not the best option

I've tested this on vanilla bungeecord and it works natively, ive also tested setting my username to be longer than 16 characters and the client is fine. I guess most of the issues would come from very old mc versions, like pre 1.8

**Comment by linsaftw** [comment]
Fixed on [1449941](https://github.com/2lstudios-mc/FlameCord/commit/144994155a6bdf2843c00a2bcd94095fecef5123)
